enacts/flex_fcst/cpt.py

```diff
@@ -106,6 +106,32 @@ def starts_list(
     return start_dates
 
 
+def read_mpycptv2dataset(data_path):
+    mu_mslices = []
+    var_mslices = []
+    obs_slices = []
+    target_count = 0
+    for targets in Path(data_path).iterdir() :
+        target_count = target_count + 1
+    for targets in Path(data_path).iterdir() :
+        new_mu, new_var, new_obs = read_pycptv2dataset(targets)
+        if target_count > 1 :
```

Comment:
In hindsight this is not a great way to distinguish dense from sparse, because there can be sparse datasets with multiple seasons. E.g. Ethiopia has two rainy seasons and we generate three starts for each. Too late to change now, we'll do it in a future version.

When generating dense forecasts we should have PyCPT put all leads for a given S in the same file. Then when reading we don't have to add L, it's already there.

Reply:
True, but in the Ethiopian case example given, we would make one app per target season and so it will be on the data manager to organize their data directory structure accordingly. I think once we have split up data cataloguing and app building, those tedious petty problems will go away.

Is it one process that generates dense forecasts? or is it multiple runs of different pyCPT configs? Because the obs aren't the same, possibly the predictor is not the same either. If so those files are not written through the same process and that would mean post-processing of the multiple files to get one... which is basically what we're doing here already (only again we're meddling what's needed for an app with what's needed to properly catalogue data). This being said, you could possibly have pyCPT generate anyways a 1-point L dim always, then cataloguing could decide whether to use it (dense case) or not